use-cases/school-timetabling/build.gradle

```
@@ -1,11 +1,11 @@
 plugins {
     id "java"
     // TODO: upgrade the Quarkus version after https://github.com/quarkusio/quarkus/issues/28591 is fixed
-    id "io.quarkus" version "2.13.0.Final"
+    id "io.quarkus" version "2.13.3.Final"
```

I think the issue is still not resolved (or didn't get into this release yet), so we must stay on Quarkus 2.13.0.Final (or provide some other workaround).

```suggestion
    id "io.quarkus" version "2.13.0.Final"
```